apollo-training/src/datasource/User.js

```diff
@@ -1,19 +1,25 @@
 import { RESTDataSource } from 'apollo-datasource-rest';
-
 import config from '../config/configurations';
 
-export class UserAPI extends RESTDataSource{
-  constructor(){
+export class UserAPI extends RESTDataSource {
+  constructor() {
     super();
     this.baseURL = `${config.serviceUrl}/api/user`;
   }
 
-  getMe(){
+  getMe() {
     return this.get('/me');
   }
 
-  loginUser(payload){
+  async loginUser(payload) {
     return this.post('/login', payload);
```

Comment:
why adding async

Reply:
because the function return promises so we are using new syntax async-await.